src/v/pandaproxy/error.h

```diff
@@ -82,6 +82,7 @@ enum class reply_error_code : uint16_t {
     schema_version_invalid = 42202,
     compatibility_level_invalid = 42203,
     subject_version_has_references = 42206,
+    subject_version_schema_id_already_exists = 42207,
```

Comment:
error code 42207 is not in confluent SR docs. In fact, they do not have error codes for 42205 or greater. Are we reserving those codes for our custom errors?
Considering the confluent SR is actively under development, I would expect them to take those error codes in the future. Should we instead take a different range of error codes? For example, 4220007 (add an extra 0 for padding).

Reply:
I discovered the error codes by manually running equivalent commands against the Confluent Schema Registry. I then duplicated the behaviour and added the tests.